### PR TITLE
Update packages & readme

### PR DESCRIPTION
**Pull request by paoliniluis**
Updating a few deps, and adding a few links to the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
-# sso-examples
+# Metabase SSO Examples
 
-Directory of Single Sign-On example applications. See the individual directories for instructions and prerequisites on running
+Directory of Single Sign-On example applications. See the individual directories for instructions.
+
+If you're looking for full app embedding, check out also our documentation in our [documentation](https://www.metabase.com/docs/latest/enterprise-guide/full-app-embedding.html)
```
Comment (walterl) [inline README.md:5]:
```suggestion
If you're looking for full app embedding, check out also our [documentation](https://www.metabase.com/docs/latest/enterprise-guide/full-app-embedding.html).
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+
+If you're looking for simple embedding, please refer to this [other repo](https://github.com/metabase/embedding-reference-apps/)
```
Comment (walterl) [inline README.md:7]:
```suggestion
If you're looking for simple embedding, please refer to this [other repo](https://github.com/metabase/embedding-reference-apps/).
```

**Review by walterl — APPROVED** [review]
Minor copy suggestions.

Just for my own edification: why was that `dns-packet` package added if no code changed?

**Comment by paoliniluis** [comment]
closing as this is super old

**Pull request closed without merging**